addons/mail/static/src/views/web/fields/emojis_field_common/emojis_field_common.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.EmojisFieldButton">
         <div class="o_mail_emojis_buttons position-relative d-inline">
             <button
-                class="btn end-0"
+                class="btn position-end-0"
```

Comment:
position-end-0 isn't a valid bootstrap class nor a custom class in Odoo, was it an unintentional change here?

Reply:
^ probably why removing position-absolute broke it @jaeschwa :p

Reply:
Yes you're right it was initially : position-absolute end-0 .